src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1956,8 +1956,13 @@ bool ApplicationManagerImpl::StartNaviService(
       /* Fix: For NaviApp1 Switch to NaviApp2, App1's Endcallback() arrives
        later than App2's Startcallback(). Cause streaming issue on HMI.
       */
-      auto accessor = applications();
-      for (auto app : accessor.GetData()) {
+      ApplicationSet apps;
```

Comment:
Maybe we could write just
`const ApplicationSet apps = applications().GetData();`
if we need a copy of apps here